terraform/testcases/prometheus_mock/ecs_taskdef.tpl

```diff
@@ -0,0 +1,101 @@
+[
```

Comment:
can you pls help rename this file to `ecs_taskdef_sidecar.tpl` so we can differentiate it with other deployment mode?

Reply:
hmm both are still sidecar deployments though right?

Reply:
the only difference here is that I changed the sample app, as prometheus sample app takes different environment variable inputs than spark sample app

Reply:
> hmm both are still sidecar deployments though right?

Yea, you sent two examples - deploy collector as sidecar with sample app, with file name `ecs_taskdef.tpl `. I mean what if in the future we want to deploy the collector as DaemonSet or a Pure service, what would be the example file name?

Reply:
I've done some example config to deploy the collector as  service, daemonset and sidecar before.